src/bundle/DependencyInjection/Configuration/Parser/FieldType/RichText.php

```diff
@@ -188,6 +191,31 @@ public function addFieldTypeSemanticConfig(NodeBuilder $nodeBuilder)
                 ->scalarPrototype()->end()
             ->end();
 
+        // RichText Toolbars configuration (defines list of Toolbars and Buttons enabled for current SiteAccess scope)
+        $nodeBuilder
+            ->arrayNode(self::TOOLBARS_NODE_KEY)
+                ->info('List of Toolbars and Buttons enabled for current SiteAccess scope.')
+                ->prototype('array')
+                    ->children()
+                        ->arrayNode('buttons')
+                            ->useAttributeAsKey('name')
+                            ->prototype('array')
+                                ->children()
+                                    ->booleanNode('visible')
+                                        ->info('Is button visible on toolbar?')
+                                        ->defaultTrue()
+                                    ->end()
+                                    ->integerNode('priority')
+                                        ->info('Defines order in which buttons appear (255 .. -255).')
```

Comment:
- 255 to 255?

Reply:
Range can be wider, but -FF and +FF is a standard for Symfony

Reply:
I understand range, I meant that `-255 to 255` is much more readable than `255 to -255` ;)

Reply:
And yet, it's incorrect from priority point of view because suggests -255 will be first when in fact will be last. :)

Reply:
least important to most important ;)